lambdas/vonage_api_ingestion/main.py

```diff
@@ -1,29 +1,22 @@
 import sys
-
 sys.path.append('./lib/')
 
 import requests
 import json
-
 from datetime import date
 from datetime import datetime
```

Comment:
consider combining these:

```suggestion
from datetime import date, datetime
```